packages/web-app-files/src/helpers/resources.js

```diff
@@ -177,11 +177,11 @@ export function buildSpace(space) {
 }
 
 export function buildWebDavFilesPath(userId, path) {
-  return `/files/${userId}/${path}`
+  return '/' + `files/${userId}/${path}`.split('/').filter(Boolean).join('/')
 }
 
 export function buildWebDavSpacesPath(spaceId, path) {
-  return `/spaces/${spaceId}/${path}`
+  return '/' + `spaces/${spaceId}/${path}`.split('/').filter(Boolean).join('/')
```

Comment:
Looking at this now it's pretty useless to put the `/` in a separate string to concatenate it :-D

Reply:
Ah, nope - it's not. The split/join afterwards removes the leading `/`